src/main/java/uk/gov/hmcts/reform/judicialapi/configuration/SecurityConfiguration.java

```diff
@@ -45,28 +46,18 @@ public class SecurityConfiguration {
 
     @Order(1)
     private ServiceAuthFilter serviceAuthFilter;
+
     @Order(2)
     private final SecurityEndpointFilter securityEndpointFilter;
-    List<String> anonymousPaths;
-
-    private JwtAuthenticationConverter jwtAuthenticationConverter;
 
-    private RestAuthenticationEntryPoint restAuthenticationEntryPoint;
+    @Setter
+    @Getter
```

Comment:
anonymousPaths does not appear to be used outside SecurityConfiguration, do we need those Lombok generated accessor methods?